static/styles/settings.css

```diff
@@ -892,13 +898,13 @@ input[type="checkbox"] {
 
 .edit_bot_email {
     font-weight: 400;
-    font-size: 18px;
+    font-size: 15px;
```

Comment:
You can just remove `font-size` from here so it can use the `font-size` from the `edit_bot_form` CSS.